### ISSUE TITLE
 `aqua list -i` is missing, whereas `aqua list --installed` exists

### ISSUE THREAD

**Issue by sato-s** [aquaproj/aqua#2952]
### Feature Overview

I'm not sure this is intentional or not.
But,  `aqua list -i` doesn't exist, whereas there is `aqua list --installed`.

I think `-i` should be defined here.
https://github.com/aquaproj/aqua/blob/3e51c0d971a5b98b4448b6c12bfbffc8e8542774/pkg/cli/list.go#L39-L48

### Why is the feature needed?

For consistency.
Because there are `aqua list -a`,  `aqua g -g`, `aqua -h`.....

### Workaround

Just use `aqua list --installed`

### Example Code

```console
$ aqua list -i # => Same output as aqua install --installed
```

**Comment by suzuki-shunsuke** [comment]
Thank you for your proposal.
It looks reasonable.
Could you send a pull request?

**Comment by sato-s** [comment]
Fixed by #2965
